src/components/com_tjcertificate/site/languages/en-GB/en-GB.com_tjcertificate.ini

```diff
@@ -52,3 +52,10 @@ COM_TJCERTIFICATE_CERTIFICATE_TEMPLATE_FIELD_PRIVATE="Private"
 COM_TJCERTIFICATE_CERTIFICATE_TEMPLATE_FIELD_PUBLIC="Public"
 COM_TJCERTIFICATE_CERTIFICATE_TEMPLATE_ENTER_CERTIFICATE_ID="Enter Certificate Id"
 
+;__DEPLOY_VERSION__
+;List view
+COM_TJCERTIFICATE_CERTIFICATE_DOWNLOAD="Download"
+COM_TJCERTIFICATE_CERTIFICATE_LIST_VIEW_TYPE="Type"
+COM_TJCERTIFICATE_CERTIFICATE_LIST_VIEW_NAME="Course Name/Event Name"
+COM_TJLMS_COURSE_TJCERT_CLIENT="Course"
+COM_JTICKETING_EVENT_TJCERT_CLIENT="Event"
```

Comment:
I know I have told you this format, but this is looking odd wrt to the rest of the lang const. Let's change this to like

`COM_TJCERTIFICATE_CLIENT_COM_TJLMS_COURSE="Course"`
`COM_TJCERTIFICATE_CLIENT_COM_JTICKETING_EVENT="Event"`

Reply:
done